src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -752,7 +756,36 @@ std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& stream, c
     OV_ITT_SCOPED_TASK(itt::domains::NPUPlugin, "Plugin::import_model");
     OV_ITT_TASK_CHAIN(PLUGIN_IMPORT_MODEL, itt::domains::NPUPlugin, "Plugin::import_model", "merge_configs");
 
-    const std::map<std::string, std::string> propertiesMap = any_copy(properties);
+    // If NPUW is active - import via NPUW
+    auto useNpuwKey = ov::intel_npu::use_npuw.name();
```

Comment:
This doesn't sound correct.

NPUW blob has its own header. So you can check if blob starts with header and that's it.
If it is not, just seek it back by the number of bytes read for the header and run the default path.

Reply:
We discussed that before. I believe there were some hidden issues with that approach. Something about benachmark_app

Reply:
what's with the benchmark_app?

Reply:
I can't remember. But done